### PR TITLE
PAL-184 helm chart for policy service

### PR DESCRIPTION
**Pull request by nw1984**

### CODE REVIEW

**Review by m09526 — CHANGES_REQUESTED** [review]
Several of the files have many mentions of the audit service, shouldn't these be policy-service? Please check the Helm names!

File: charts/policy-service/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: {{ include "audit-service.name" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/deployment.yaml]: oops!

File: charts/policy-service/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    namespace: {{ include "palisade.namespace" . }}
+    labels:
+            {{ include "audit-service.labels" . | indent 4 }}
```
Comment (dev-958) [inline charts/policy-service/templates/deployment.yaml]: oops!

File: charts/policy-service/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+spec:
+    replicas: {{ .Values.replicaCount }}
+    selector:
+        matchLabels:
+            app.kubernetes.io/name: {{ include "audit-service.name" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/deployment.yaml]: oops!

File: charts/policy-service/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            app.kubernetes.io/instance: {{ .Release.Name }}
+    template:
+        metadata:
+            labels:
+                app.kubernetes.io/name: {{ include "audit-service.name" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/deployment.yaml]: oops!

File: charts/policy-service/templates/service.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+    name: {{ include "audit-service.name" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/service.yaml]: oops!

File: charts/policy-service/templates/service.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        {{ include "audit-service.labels" . | indent 4 }}
+spec:
+    type: {{ .Values.service.type }}
+    ports:
+    -   port: {{ .Values.service.port }}
+        targetPort: 8085
+        protocol: TCP
+        name: web
+    selector:
+        app.kubernetes.io/name: {{ include "audit-service.name" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/service.yaml]: oops!

**Review by dev-958 — CHANGES_REQUESTED** [review]
See inline. Stop hurrying! No prizes for tickets.

**Review by m09526 — DISMISSED** [review]

File: charts/policy-service/templates/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+# limitations under the License.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: {{ include "policy-service.name" . }}
+    namespace: {{ include "palisade.namespace" . }}
+    labels:
+        app.kubernetes.io/name: {{ include "palisade-service.name" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/deployment.yaml]: nope

File: charts/policy-service/templates/deployment.yaml [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+        helm.sh/chart: {{ include "palisade-service.chart" . }}
```
Comment (dev-958) [inline charts/policy-service/templates/deployment.yaml]: nope

**Review by dev-958 — CHANGES_REQUESTED** [review]
see inline

**Review by dev-958 — APPROVED** [review]

**Review by m09526 — APPROVED** [review]

**Pull request merged**